packages/salesforcedx-vscode-apex/src/languageServer.ts

```diff
@@ -64,11 +65,9 @@ async function createServer(
     if (jvmMaxHeap) {
       args.push(`-Xmx${jvmMaxHeap}M`);
     }
-    telemetryService.sendEventData(
-      'apexLSPSettings',
-      undefined,
-      { maxHeapSize: jvmMaxHeap != null ? jvmMaxHeap : 0 }
-    );
+    telemetryService.sendEventData('apexLSPSettings', undefined, {
```

Comment:
@klewis-sfdc was this formatting change intentional? I thought it was a bit easier to read before

Reply:
No, it just happened when I saved the file.  Our formatter thinks this is the correct way I guess